packages/features/ee/teams/components/MemberListItem.tsx

```diff
@@ -546,137 +571,101 @@ export default function MemberListItem(props: Props) {
-          const isVisible = numberOfSelectedRows >= 2 && dynamicLinkVisible;
-
-          const users = table
-            .getSelectedRowModel()
-            .flatRows.map((row) => row.original.username)
-            .filter((u) => u !== null);
-
-          const usersNameAsString = users.join("+");
-
-          const dynamicLinkOfSelectedUsers = `${domain}/${usersNameAsString}`;
-          const domainWithoutHttps = dynamicLinkOfSelectedUsers.replace(/https?:\/\//g, "");
-
-          return (
-            <>
-              {isVisible ? (
-                <m.div
-                  layout
-                  className="bg-brand-default text-inverted item-center animate-fade-in-bottom hidden w-full gap-1 rounded-lg p-2 text-sm font-medium leading-none md:flex">
-                  <div className="w-[300px] items-center truncate p-2">
-                    <p>{domainWithoutHttps}</p>
-                  </div>
-                  <div className="ml-auto flex items-center">
-                    <Button
-                      StartIcon="copy"
-                      size="sm"
-                      onClick={() => copyToClipboard(dynamicLinkOfSelectedUsers)}>
-                      {!isCopied ? t("copy") : t("copied")}
-                    </Button>
-                    <Button
-                      EndIcon="external-link"
-                      size="sm"
-                      href={dynamicLinkOfSelectedUsers}
-                      target="_blank"
-                      rel="noopener noreferrer">
-                      Open
-                    </Button>
-                  </div>
-                </m.div>
-              ) : null}
-            </>
-          );
-        }}
+        data-testid="user-list-data-table"
+        table={table}
         tableContainerRef={tableContainerRef}
-        onScroll={(e) => fetchMoreOnBottomReached(e.target as HTMLDivElement)}
-        tableCTA={
-          isAdminOrOwner || props.isOrgAdminOrOwner ? (
-            <Button
-              type="button"
-              color="primary"
-              StartIcon="plus"
-              size="sm"
-              className="rounded-md"
-              onClick={() => props.setShowMemberInvitationModal(true)}
-              data-testid="new-member-button">
-              {t("add")}
-            </Button>
-          ) : (
-            <></>
-          )
-        }
-        columns={memorisedColumns}
-        data={flatData}
         isPending={isPending}
-        filterableItems={[
-          {
-            tableAccessor: "role",
-            title: "Role",
-            options: [
-              { label: "Owner", value: "OWNER" },
-              { label: "Admin", value: "ADMIN" },
-              { label: "Member", value: "MEMBER" },
-              { label: "Pending", value: "PENDING" },
-            ],
-          },
-        ]}
-      />
-
+        onScroll={(e) => fetchMoreOnBottomReached(e.target as HTMLDivElement)}>
+        <DataTableToolbar.Root>
```

Comment:
Added composable components instead of using props which inevitably made this whole data component to expand outside of the generic usecase that we had when it was first created